codex-ui/src/vue/editor/useEditor.ts

```diff
@@ -36,20 +32,26 @@ interface UseEditorParams {
    * Handles content change in Editor
    */
   onChange?: (data: OutputData) => void;
+
+  /**
+   * Loaded user tools for Editor
+   */
+  tools: DownloadedTools;
 }
 
 /**
  * Handles Editor.js instance creation
  *
  * @param params - Editor.js params
  */
-export function useEditor({ id, content, isReadOnly, onChange }: UseEditorParams): {
+export function useEditor({ id, content, isReadOnly, onChange, tools }: UseEditorParams): {
   isEmpty: Ref<boolean>;
   refresh: (data: OutputData) => void;
 } {
   /**
    * Editor instance
    */
+  // @ts-expect-error Ts config doesn't match with Editor
```

Comment:
We have different `tsconfig` from main. And if i change config in `tsconfig.json`, i have some errors in different ui component. So i have question, we change `tsconfig` and try to fix errors ?

Reply:
what error does it display here?